Procfile

```diff
@@ -1 +1,2 @@
 web: bundle exec puma -p $PORT
+worker: bundle exec sidekiq -C config/sidekiq.yml
```

Comment:
add new empty line @ end of file...

Reply:
done https://github.com/DFE-Digital/register-trainee-teacher-data/pull/30/commits/2303128560223e4f2c2c4c9c241b5e28f4afd8bf